lib/toolbar/edit-toolbar.vue

```diff
@@ -405,7 +407,6 @@
           const activeNav = savedTab || 'all-pages';
 
           this.$store.dispatch('showNavBackground', true);
-  
```

Comment:
Please leave an empty line before a `return` statement to make ir more readable

Reply:
ohhhh there was an empty space that my linter removed. Thank you man.